css/index.css

```diff
@@ -143,4 +141,168 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+/* Sprint Code Here */
+
+header {
+    width: 800px;
+    margin: 40px auto;
+}
+
+nav {
+    display: flex;
+    align-items: baseline;
+    justify-content: space-between;
+    margin-bottom: 40px;
+}
+
+nav .navbar {
+    width: 80%;
+    display: flex;
+    align-items: flex-end;
+    justify-content: space-around;
+}
+
+.navbar a {
+    text-decoration: none;
+    color: black;
+}
+
+.navbar a:hover {
+    text-decoration: underline;
+}
+
+/* Superior glorious 24-bit color box styling */
+
+.box:nth-child(1) {
```

Comment:
Nth child is the perfect solution when you don't think you'll need to reuse the styles elsewhere which makes it ideal here.